### ISSUE TITLE
Master password field becomes unresponsive indefinitely after Password was entered incorrect once Firefox/MaxOS

### ISSUE THREAD

**Issue by mxbchr** [bitwarden/clients#6386]
### Steps To Reproduce

1. Install and connect Firefox Bitwarden Addon
2. Type in wrong Master-Password to unlock Vault, see the wrong password message
3. Type in the correct Password, hit enter - nothing happens

### Expected Result

After typing in the correct master-password, the vault should unlock normally.

### Actual Result

Master-Password field becomes unresponsive. Neither hitting enter nor clicking "unlock" triggers the password check/unlock process. 
This is indefinitely, only a browser restart resets the issue.
The Addon was working fine until a few days ago.

### Operating System

macOS

### Operating System Version

12.6

### Web Browser

Firefox

### Browser Version

117.0.1 (64-bit)

### Build Version

2023.8.3

### Issue Tracking Info

- [x] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by crespire** [comment]
I've run into this as well. Same browser and build version, but on Windows 10 Pro, 22H2, build 19045.3448.

**Comment by CodingCellist** [comment]
I am seeing the same behaviour on FF Nightly 119.0a1 (2023-09-24) (64-bit), on both Windows 10 and Linux.

In case it helps, here's the FF add-on debug output:

<details>

<summary><code>Uncaught TypeError: can't access dead object [...]</code></summary>

```
Uncaught TypeError: can't access dead object
    state moz-extension://<uuid>/popup/main.js:1
    TS moz-extension://<uuid>/popup/main.js:1
    N moz-extension://<uuid>/popup/polyfills.js:2
    TS moz-extension://<uuid>/popup/main.js:1
    state moz-extension://<uuid>/popup/main.js:1
    defaultInMemoryOptions moz-extension://<uuid>/popup/main.js:1
    TS moz-extension://<uuid>/popup/main.js:1
    N moz-extension://<uuid>/popup/polyfills.js:2
    TS moz-extension://<uuid>/popup/main.js:1
    defaultInMemoryOptions moz-extension://<uuid>/popup/main.js:1
    getUserKey moz-extension://<uuid>/popup/main.js:1
    TS moz-extension://<uuid>/popup/main.js:1
    N moz-extension://<uuid>/popup/polyfills.js:2
    TS moz-extension://<uuid>/popup/main.js:1
    getUserKey moz-extension://<uuid>/popup/main.js:1
    ES moz-extension://<uuid>/popup/main.js:1
    TS moz-extension://<uuid>/popup/main.js:1
    N moz-extension://<uuid>/popup/polyfills.js:2
    TS moz-extension://<uuid>/popup/main.js:1
    ES moz-extension://<uuid>/popup/main.js:1
    y moz-extension://<uuid>/popup/vendor.js:2
    f moz-extension://<uuid>/popup/vendor.js:2
    _next moz-extension://<uuid>/popup/vendor.js:2
    next moz-extension://<uuid>/popup/vendor.js:2
    next moz-extension://<uuid>/popup/vendor.js:2
    n moz-extension://<uuid>/popup/vendor.js:2
    next moz-extension://<uuid>/popup/vendor.js:2
    next moz-extension://<uuid>/popup/vendor.js:2
    update moz-extension://<uuid>/popup/main.js:1
    t_ moz-extension://<uuid>/popup/main.js:1
    N moz-extension://<uuid>/popup/polyfills.js:2
    t_ moz-extension://<uuid>/popup/main.js:1
    update moz-extension://<uuid>/popup/main.js:1
    updateFromMessage moz-extension://<uuid>/popup/main.js:1
    t_ moz-extension://<uuid>/popup/main.js:1
    N moz-extension://<uuid>/popup/polyfills.js:2
    t_ moz-extension://<uuid>/popup/main.js:1
    updateFromMessage moz-extension://<uuid>/popup/main.js:1
    listenForUpdates moz-extension://<uuid>/popup/main.js:1
    t_ moz-extension://<uuid>/popup/main.js:1
    N moz-extension://<uuid>/popup/polyfills.js:2
    t_ moz-extension://<uuid>/popup/main.js:1
    listenForUpdates moz-extension://<uuid>/popup/main.js:1
main.js:1:1128221
```

</details>

**Comment by SergeantConfused** [comment]
Hello @mxbchr and everyone else,

Thank you for reporting this. I tested this on Windows 11 Pro with Firefox 117.0.1 and Bitwarden Add-on 2023.8.3, and I am unable to reproduce this; I installed the extension, logged into an account, closed the browser, opened the browser, hit CTRL+SHIFT+Y and typed an incorrect master password and received an error, and afterwards I pressed CTRL+A and typed the correct master password and hit Enter (Return) and I was able to unlock the extension. I recommend performing a clean new installation; Could you please uninstall your extensions and install them anew, and test that once more?

If this behaviour persists, please create a support ticket (https://bitwarden.com/help/) so we'd have a look at this together, and please also include a link to this GitHub report.

Alternatively, you can also seek assistance from other Bitwarden users in our community forums (https://community.bitwarden.com/c/support/), if you wish.

We use GitHub issues as a place to track bugs and other development related issues; The issue here will be closed.


Thank you again,

**Comment by mxbchr** [comment]
@SergeantConfused 
I uninstalled the extension and reinstalled them. Error persists.
Gonna open a ticket as suggested.

**Comment by mxbchr** [comment]
I downgraded the addon to Version 2023.7.1, now everything works fine again.
So it is definitely the update to Version 2023.8.3 that broke it.

To donwgrade the addon, got to:
https://addons.mozilla.org/en-US/firefox/addon/bitwarden-password-manager/versions/?utm_content=search&utm_medium=referral&utm_source=addons.mozilla.org

Scroll down to 2023.7.1  and click “Download File” next to it and then install when prompted.
This just overwrites the addon with the older version and retains all configuration.

**Comment by SergeantConfused** [comment]
Hi @mxbchr,

Could you please let me know if you still are encountering this in version 2023.9.1 of our Firefox Add-on?


Thank you in advance,

**Comment by mxbchr** [comment]
@SergeantConfused 

2023.9.1 resolves the issue. Everything works fine now again.

Thank you!

**Comment by CodingCellist** [comment]
@SergeantConfused can also confirm that 2023.9.1 fixes things on the latest FF nightly (120.0a1). Cheers!
